src/modules/hunting/cvehunter.py

```diff
@@ -32,7 +32,9 @@ def __init__(self, evidence):
 # Passive Hunter
 @handler.subscribe(ApiServer)
 class IsVulnerableToCVEAttack(Hunter):
-    """ Node is running a Kubernetes version vulnerable to critical CVE-2018-1002105 """
+    """API server version end point and CVE-2018-1002105 hunter
```

Comment:
I am not sure that a comment this long is going to be readable in the output table. Have you checked this running the regular text output, not just the json output?

When you're changing the output please include a "before" and "after" copy of the output so reviewers can see what's changing.

Reply:
I still prefer the original text for this, what was the reason to change it?